automate-mautic.php

```diff
@@ -1,26 +1,19 @@
 <?php
 /**
- * Plugin Name: AutomatePlus - Mautic for WordPress
+ * Plugin Name: AutomatePlug - Mautic for WordPress
  * Plugin URI: http://www.brainstormforce.com/
  * Description: Add registered WP users and commentors to mautic contacts.
- * Version: 1.0.4
+ * Version: 1.0.5
  * Author: Brainstorm Force
  * Author URI: http://www.brainstormforce.com/
- * Text Domain: automateplus-mautic-wp
+ * Text Domain: automateplug-mautic-wp
```

Comment:
I don't think we can change the text domain at this moment, since the text domain should be same as the plugin slug and we cannot change the plugin slug